src/ftest.jl

```diff
@@ -56,8 +56,8 @@ this is an ANOVA, our null hypothesis is that `Result ~ 1` fits the data as well
 
 ```jldoctest
 julia> dat = DataFrame(Treatment=[1, 1, 1, 2, 2, 2, 1, 1, 1, 2, 2, 2.],
-                       Result=[1.1, 1.2, 1, 2.2, 1.9, 2, .9, 1, 1, 2.2, 2, 2],
-                       Other=[1, 1, 2, 1, 2, 1, 3, 1, 1, 2, 2, 1]);
+                              Result=[1.1, 1.2, 1, 2.2, 1.9, 2, .9, 1, 1, 2.2, 2, 2],
```

Comment:
Doesn't this break alignment?

Reply:
It doesn't. I tried to literally copy the REPL output to see if that helped. I'll revert to the aligned version.